source/_integrations/hydrawise.markdown

```diff
@@ -66,10 +66,10 @@ monitored_conditions:
   type: list
   default: All binary sensors are enabled.
   keys:
-    is_watering:
+    watering:
       description: The binary sensor is `on` when the zone is actively watering.
-    status:
-      description: This will indicate `on` when there is a connection to the Hydrawise cloud. It is not an indication of whether the irrigation controller hardware is online.
+    my_controller_status:
```

Comment:
This seems incorrect. According to the code it validates a list of binary sensors here:

<https://github.com/home-assistant/core/blob/6023105c6a3b69ffb2338c494246802c086aaee4/homeassistant/components/hydrawise/binary_sensor.py#L17>

While the list itself is here:

```py
BINARY_SENSORS = ["is_watering", "status"]
```

<https://github.com/home-assistant/core/blob/6023105c6a3b69ffb2338c494246802c086aaee4/homeassistant/components/hydrawise/__init__.py#L57>